src/data/Graph.js

```diff
@@ -137,9 +137,10 @@ graphProto.getNodeById = function (id) {
  * @param {number|string|module:echarts/data/Graph.Node} n1
  * @param {number|string|module:echarts/data/Graph.Node} n2
  * @param {number} [dataIndex=-1]
+ * @param forceIgnore
  * @return {module:echarts/data/Graph.Edge}
  */
-graphProto.addEdge = function (n1, n2, dataIndex) {
+graphProto.addEdge = function (n1, n2, dataIndex, forceIgnore) {
```

Comment:
The data used in forceHelper is initialized 

https://github.com/apache/incubator-echarts/blob/master/src/chart/graph/forceLayout.js#L90

There is no need to store this `ignoreFoceLayout` property in the Graph structure.